src/App.js

```diff
@@ -1,27 +1,10 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
```

Comment:
- [ ] So far so good. All components are supposed to be class-based as directed by the requirements though. Please refactor this to be a class-based component. Take a look at that requirement below:

---
> - Use class based components.
---